paddlenlp/trainer/training_args.py

```diff
@@ -583,6 +587,15 @@ class TrainingArguments:
             )
         },
     )
+    cp_parallel_degree: int = field(
```

Comment:
换成 `context_parallel_degree` ？
```suggestion
    context_parallel_degree: int = field(
```